README.md

```diff
@@ -33,13 +33,10 @@ Try the autocomplete and the instantsearch results page on our [live demo](https
 
 We are happy to announce that version 3.x of our extension is now compatible with Magento 2.4. Review the [Customisation](https://github.com/algolia/algoliasearch-magento-2#customisation) section to learn more about the differences between our extension versions.
 
-The 1.x and 2.x versions of the extension are now End of Life (EOL). We no longer accept community PRs for those branches. 
-
-| Extension Version | End of Life |
-| --- | --- |
-| v1.x | Dec 2020 |
-| v2.x | Dec 2020 |
-| v3.x | N/A |
+| Extension Version | End of Life | Magento  | PHP |
+|-------------------| --- |----------| ---------|
```

Comment:
Let's add entries between `3.7.x` and `3.13.x` after conducting tests against Magento / PHP (this is for a future separate PR)